src/Symfony/Component/Debug/FatalErrorHandler/UndefinedMethodFatalErrorHandler.php

```diff
@@ -36,8 +36,12 @@ public function handleError(array $error, FatalErrorException $exception)
 
         $message = sprintf('Attempted to call an undefined method named "%s" of class "%s".', $methodName, $className);
 
+        if (null === $methods = get_class_methods($className)) {
```

Comment:
looking at the comments below, this could be turned to: `if (!class_exists($className, false))`, isn'it?